version.dd

```diff
@@ -289,21 +289,21 @@ $(H3 $(LEGACY_LNAME2 PredefinedVersions, predefined-versions, Predefined Version
 	$(TROW $(ARGS $(D Alpha_HardFloat)) , $(ARGS The Alpha hard float ABI))
 	$(TROW $(ARGS $(D LittleEndian)) , $(ARGS Byte order, least significant first))
 	$(TROW $(ARGS $(D BigEndian)) , $(ARGS Byte order, most significant first))
-	$(TROW $(ARGS $(D D_Coverage)) , $(ARGS $(DPLLINK code_coverage.html, Code coverage analysis) instrumentation (command line $(DDSUBLINK dmd-windows, switches, switch) $(D -cov)) is being generated))
-	$(TROW $(ARGS $(D D_Ddoc)) , $(ARGS $(DDLINK ddoc, Embedded Documentation, Ddoc) documentation (command line $(DDSUBLINK dmd-windows, switches, switch) $(D -D)) is being generated))
-	$(TROW $(ARGS $(D D_Warnings)) , dmd warnings are activated (command line $(DDSUBLINK dmd-windows, switches, switch) $(D -w) or $(D -wi)))
+	$(TROW $(ARGS $(D D_Coverage)) , $(ARGS $(DPLLINK code_coverage.html, Code coverage analysis) instrumentation (command line $(DDSUBLINK dmd-windows, switch-cov, $(TT -cov))) is being generated))
```

Comment:
You dropped the word "switch" throughout. Not sure if that's deliberate or if it's a mistake. To me, "command line switch -cov" makes more sense than "command line -cov".